test/config.test.js

```diff
@@ -67,8 +74,8 @@ describe('Config migration', () => {
   });
 
   it('does not migrate when @percy/cli is not installed', async () => {
-    mockRequire('fs', { existsSync: () => false });
-    await Migrate('--only-cli');
+    spyOn(fs, 'existsSync').and.callFake(p => p.endsWith('package.json'));
```

Comment:
Alternatively, we could just delete this file file from the mock filesystem without spying on fs. Especially since it's already spied with a specific implementation that this would override (so we should avoid spying on fs when possible and just rely on the mock filesystem for fs things).